src/workflows/test_workflows/annotation/scgpt_knn_test/script.py

```diff
@@ -0,0 +1,40 @@
+from mudata import read_h5mu
+import shutil
+import os
+import sys
+from pathlib import Path
+import pytest
+
+##VIASH START
+par = {
+    "input": "input.h5mu"
+}
+
+meta = {
+    "resources_dir": "resources_test"
+}
+##VIASH END
+
+
+def test_run():
```

Comment:
Ideally, the test should also take in the the original input file to check if the data from the original input mudata file stay intact. I think we do something similar in the integration tests from `process_batches`. Do you think that i feasible here?

Reply:
Also true!